components/style/gecko_bindings/mod.rs

```diff
@@ -2,24 +2,5 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
```

Comment:
This file should be unchanged. `check_bindings.rs` is not actually going to be used in this crate, so you don't need a `mod` for it